src/server/shttpd/io_emb.c

```diff
@@ -306,7 +306,7 @@ shttpd_wakeup(const void *priv)
 	conn->flags &= ~SHTTPD_SUSPEND;
 #endif
 	(void) memcpy(buf, &cmd, sizeof(cmd));
-	(void) memcpy(buf + sizeof(cmd), conn, sizeof(conn));
+	(void) memcpy(buf + sizeof(cmd), &conn, sizeof(conn));
```

Comment:
Hmm, this looks wrong. conn is declared as `const struct conn       *`, so it is already a pointer and the original `memcpy` call looks right to me.

Reply:
The `sizeof` call needs to be changed instead and read `sizeof(struct conn)`

Reply:
Yep, the fix look wrong now, but I think I know why static analyzer jumped: sizeof(conn) is sizeof of pointer, not the structure, sod sizeof(conn) -> sizeof(struct conn) should help here

Reply:
Or better rename conn pointer to something else to avoid name collision.

Reply:
I hate `shttpd` :stuck_out_tongue_winking_eye: